build.sbt

```diff
@@ -33,15 +33,16 @@ val zioVersion = "2.0.15"
 lazy val root = project
   .in(file("."))
   .settings(
-    skip / publish := true,
+    publish / skip := true,
```

Comment:
Reference: https://www.scala-sbt.org/1.x/docs/Publishing.html#Skip+publishing
The effect of this line is that `sbt publish` or `sbt publishLocal` would not generate an artifact named `root`, which it currently does.